fv3core/stencils/basic_operations.py

```diff
@@ -39,10 +39,15 @@ def copy(q_in, origin=(0, 0, 0), domain=None):
     return q_out
 
 
+@gtscript.function
+def adjustmentfactor(adjustment: sd, q_out: sd):
+    return q_out * adjustment
+
+
 @gtstencil()
 def adjustmentfactor_stencil(adjustment: sd, q_out: sd):
     with computation(PARALLEL), interval(...):
-        q_out[0, 0, 0] = q_out * adjustment
+        q_out[0, 0, 0] = adjustmentfactor(adjustment, q_out)
```

Comment:
Is this a pattern we're moving to now? Otherwise I'm not sure why to put the calculation in a function vs. leaving it in a stencil.

Reply:
it's mostly because I made a function, so I might as well call it, and I was thinking we'd probably eventually delete the stencil version of it. But, I can undo this, it does a layer of obscurity to it.

Reply:
I was mostly curious, I don't think you need to revert this